src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -584,23 +584,105 @@ private static int GetFloatingPointMaxDigitsAndPrecision(char fmt, ref int preci
             return null;
         }
 
+        public static string FormatHalf(Half value, string? format, NumberFormatInfo info)
```

Comment:
nit: This should probably be just have `TryFormatHalf` so they are grouped together (like is done for `Single` and `Double`)

Reply:
Hmmm, not sure I follow. There is also a `TryFormatHalf` method below. Line 608

Reply:
Sorry, this was meant to say `just above`. That is, `FormatSingle` and `TryFormatSingle` are grouped together, as are `FormatDouble` and `TryFormatDouble`.

`FormatHalf`, however, isn't grouped with `TryFormatHalf`